doc/index.rst

```diff
@@ -151,15 +151,16 @@ To see the full config from this bundle, run:
 
     $ php bin/console config:dump symfonycasts_tailwind
 
-The main option is ``input_css`` option, which defaults to ``assets/styles/app.css``.
-This represents the "source" Tailwind file (the one that contains the ``@tailwind``
+The main option is ``input`` option, which defaults to ``assets/styles/app.css``.
+This represents the "source" Tailwind files (the one that contains the ``@tailwind``
 directives):
 
 .. code-block:: yaml
 
     # config/packages/symfonycasts_tailwind.yaml
     symfonycasts_tailwind:
-        input_css: 'assets/styles/other.css'
+        input:
```

Comment:
Hm, is it accurate? I see you kept the `input_css` option in the code, so it seems docs should be reverted?

Reply:
Oh yes. I forgot to revert.
I changed it while making the test for the implementation. I'll revert it and submit again